spring-xd-shell/src/main/java/org/springframework/xd/shell/command/JobCommands.java

```diff
@@ -358,6 +360,9 @@ public String deployJob(
 				}
 				propertiesToUse = DeploymentPropertiesFormat.convert(props);
 				break;
+			case -1: // Neither option specified
+				propertiesToUse = Collections.EMPTY_MAP;
```

Comment:
I think `propertiesToUse = Collections.emptyMap();` would work too.

In any case, nice catch David, thx